src/DssSpell.sol

```diff
@@ -16,116 +16,66 @@
-            ilkStabilityFee:       ONE_PCT_RATE,
-            startingPriceFactor:   10500,
-            breakerTolerance:      9500, // Allows for a 5% hourly price drop before disabling liquidations
-            auctionDuration:       220 minutes,
-            permittedDrop:         9000,
-            liquidationRatio:      10500,
-            kprFlatReward:         300,
-            kprPctReward:          10 // 0.1%
-        });
-
-        DssExecLib.addNewCollateral(GUNIV3DAIUSDC1_A);
-        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 10 * MILLION, 10 * MILLION, 8 hours);
-
-        DssExecLib.setChangelogAddress("GUNIV3DAIUSDC1", GUNIV3DAIUSDC1);
-        DssExecLib.setChangelogAddress("MCD_JOIN_GUNIV3DAIUSDC1_A", MCD_JOIN_GUNIV3DAIUSDC1_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_GUNIV3DAIUSDC1_A", MCD_CLIP_GUNIV3DAIUSDC1_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_GUNIV3DAIUSDC1_A", MCD_CLIP_CALC_GUNIV3DAIUSDC1_A);
-        DssExecLib.setChangelogAddress("PIP_GUNIV3DAIUSDC1", PIP_GUNIV3DAIUSDC1);
-        DssExecLib.setChangelogVersion("1.9.6");
+        // Adjusting Auction Parameters for ETH-A, ETH-B, ETH-C, and WBTC-A
+        // https://vote.makerdao.com/polling/QmfGk3Dm?network=mainnet#poll-detail
+
+        // Decrease the Auction Price Multiplier (buf) for ETH-A, ETH-C, and WBTC-A vaults from 1.3 to 1.2
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-A",  12000);
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-C",  12000);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-A", 12000);
+
+        // Increase the Local Liquidation Limit (ilk.hole)
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-A",  40 * MILLION); //  from 30 Million DAI to 40 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-B",  25 * MILLION); //  from 15 Million DAI to 25 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-C",  30 * MILLION); //  from 20 Million DAI to 30 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-A", 25 * MILLION); //  from 15 Million DAI to 25 Million DAI
+
+        // Perform a Test Spell on Optimism
+        OptimismL1GovernanceRelayLike(OPTIMISM_L1_GOVERNANCE_RELAY).relay(
+            OPTIMISM_L2_SPELL,                /* target */
+            abi.encodeWithSignature("act()"), /* targetData */
+            3000000                           /* l2gas */
+        );
+
+        // Perform a Test Spell on Arbitrum
+        ArbitrumL1GovernanceRelayLike(ARBITRUM_L1_GOVERNANCE_RELAY).relay(
+            ARBITRUM_L2_SPELL,                /* target */
+            abi.encodeWithSignature("act()"), /* targetData */
+            680168429264,                     /* l1CallValue */
+            0,                                /* maxGas */
+            0,                                /* gasPriceBid */
+            680168429264                      /* maxSubmissionCost */
+        );
```

Comment:
@krzkaczor I read your comment in the other PR that the Arbitrum callvalue and max submission values need to be updated with the most recent ones before deploying. Is there any risk that the spell can revert due a wrong param in that moment?
If there is a minimum risk of so we should encapsulate it so it doesn't make the whole spell to revert.

Reply:
No, in the worst-case L2 spell won't fire and it will require another L1 spell.

Reply:
perfect, thanks